tests/PluginTest.php

```diff
@@ -35,8 +35,28 @@ public function testGetSubscribedEvents()
         $this->assertInternalType('array', $plugin->getSubscribedEvents());
     }
 
+    /**
+    * Tests handleTableflipCommand().
+    */
     public function testHandleTableflipCommand()
     {
+        $event = $this->getMockCommandEvent();
+        Phake::when($event)->getCustomParams()->thenReturn(array());
+        Phake::when($event)->getSource()->thenReturn('#channel1');
+        Phake::when($event)->getCommand()->thenReturn('PRIVMSG');
+        $queue = $this->getMockEventQueue();
+
+        $plugin = new Plugin;
+        $plugin->handleTableflipCommand($event, $queue);
+
+        Phake::when($event)->getCustomParams()->thenReturn(array('#channel1', 'tableflip some text'));
+        $plugin->handleTableflipCommand($event, $queue);
+
+        Phake::inOrder(
```

Comment:
There was 1 failure:

1) Phergie\Irc\Tests\Plugin\React\TableFlip\PluginTest::testHandleTableflipCommand
Failed asserting that Expected Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(equal to <string:#channel1>, equal to <array>) to be called at least <1> times, actually called <0> times.
Other Invocations:
  Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(<string:#channel1>, <null>).

/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/vendor/phake/phake/src/Phake/Client/PHPUnit.php:56
/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/vendor/phake/phake/src/Phake/Proxies/VerifierProxy.php:109
/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/tests/PluginTest.php:56
/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/tests/PluginTest.php:56

FAILURES!
Tests: 3, Assertions: 3, Failures: 1.